tests/test_pagination.py

```diff
@@ -331,6 +348,62 @@ def test_relay_cursor_sorted(client, data):
     }
 
 
+def test_relay_cursor_sorted_none_desc(client, data_with_nulls):
+    response = client.get("/relay_cursor_widgets?sort=-name")
+    assert_response(
+        response, 200, [{"id": "8", "name": None}, {"id": "7", "name": None}]
+    )
+    cursor = get_meta(response)["cursors"][-1]
+    response = client.get(f"/relay_cursor_widgets?sort=-name&cursor={cursor}")
+    assert_response(
+        response,
+        200,
+        [{"id": "4", "name": None}, {"id": "1", "name": "Whatzit"}],
+    )
+    cursor = get_meta(response)["cursors"][-1]
+    response = client.get(f"/relay_cursor_widgets?sort=-name&cursor={cursor}")
+    assert_response(
+        response,
+        200,
+        [{"id": "3", "name": "Plus Ultra"}, {"id": "5", "name": "Fooz"}],
+    )
+    cursor = get_meta(response)["cursors"][-1]
+    response = client.get(f"/relay_cursor_widgets?sort=-name&cursor={cursor}")
+    assert_response(
+        response,
+        200,
+        [{"id": "6", "name": "Doodad"}, {"id": "2", "name": "AAA Time"}],
+    )
+
+
+def test_relay_cursor_sorted_none_asc(client, data_with_nulls):
+    response = client.get("/relay_cursor_widgets?sort=name")
+    assert_response(
+        response,
+        200,
+        [{"id": "2", "name": "AAA Time"}, {"id": "6", "name": "Doodad"}],
+    )
+    cursor = get_meta(response)["cursors"][-1]
+    response = client.get(f"/relay_cursor_widgets?sort=name&cursor={cursor}")
+    assert_response(
+        response,
+        200,
+        [{"id": "5", "name": "Fooz"}, {"id": "3", "name": "Plus Ultra"}],
+    )
+    cursor = get_meta(response)["cursors"][-1]
+    response = client.get(f"/relay_cursor_widgets?sort=name&cursor={cursor}")
+    assert_response(
+        response,
+        200,
+        [{"id": "1", "name": "Whatzit"}, {"id": "4", "name": None}],
+    )
+    cursor = get_meta(response)["cursors"][-1]
+    response = client.get(f"/relay_cursor_widgets?sort=name&cursor={cursor}")
```

Comment:
can we also test with backwards pagination? eg. the `before` arg. Shouldn't cause any issues, but worth covering